common/changes/master_2017-02-28-15-05.json

```diff
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Pass index, hasCheckMarks, hasIcons to Contextmual menu item onRender",
+      "type": "patch"
```

Comment:
nit: typo in `ContextMenu`. Also start with component name followed by fix details. Thanks